include/mxnet/mxruntime.h

```diff
@@ -0,0 +1,194 @@
+        /*
```

Comment:
space. also why not name it with mx prefix? I don't think this convention is used anywhere else.

Reply:
there is another files in the project named runtime.h so there's a clash with the includes. Naming it runtime.h was prevented by the IDE. Do you have a better suggestion? I don't have a strong opinion on this.